src/components/panel/panel.js

```diff
@@ -832,6 +850,47 @@ MdPanelRef.prototype._addStyles = function() {
 };
 
 
+/**
+ * Add ARIA-specific attributes appropriate for panels.
+ */
+MdPanelRef.prototype._configureAria = function() {
```

Comment:
nit: "ARIA" refers to the extra attributes on top of html that screen readers understand. Focus trapping is unrelated to ARIA; it just _also_ improves accessibility. I would just call `configureFocusTrap` directly (and set the `tabindex` there).